40196.0.txt

```diff
@@ -0,0 +1,97 @@
+@0
+
+
+@1
+#Pass two values down and move one to acc
+MOV UP, DOWN
+MOV UP, DOWN
+A:
+MOV UP, ACC
+
+# Check if its 0
+JNZ FALSE
+#If it is 0, send 1 right
+MOV 1, RIGHT
+JMP B
+
+# Otherwise send 0 right
+FALSE:
+MOV 0, RIGHT
+
+B:
+MOV ACC, DOWN
+JMP A
+
+@2
+MOV LEFT, DOWN
+
+@3
+
+
+@4
+#Send one value on and move other to acc
+MOV UP, DOWN
+
+A:
+MOV UP, ACC
+
+#Check if its 0
+JNZ FALSE
+
+#if 0 move 1 right
+MOV 1, RIGHT
+JMP B
+
+#else move 0 right
+FALSE:
+MOV 0, RIGHT
+
+B:
+MOV ACC, DOWN
+JMP A
+
+@5
+MOV LEFT, ACC
+ADD UP
+MOV ACC, DOWN
+
+@6
+
+
+@7
+
+
+@8
+#Repeat checks for if its 0, send 1 right if it is, 0 if it isnt
+A:
+MOV UP, ACC
+
+JNZ FALSE
+MOV 1, RIGHT
+JMP A
+
+FALSE:
+MOV 0, RIGHT
+
+@9
+#Collect the 3 values and add them together
+MOV 0, DOWN
+MOV 0, DOWN
+
+A:
+MOV UP, ACC
+ADD LEFT
+#Subtract 3 and check if result = 0 If sequence was 0, 0, 0, values should have been 1+1+1 = 3 leaving 0 when 3 is subtracted.
```

Comment:
Comments on later levels are good overall -- good focus on explaining "why" rather than "what". Keep it up!